lib/tasks/rotate.rake

```diff
@@ -11,9 +11,13 @@ namespace :rotate do
     User.find_in_batches.with_index do |users, _batch|
       User.transaction do
         users.each do |user|
-          rotator = KeyRotator::AttributeEncryption.new(user)
-          rotator.rotate
-          progress&.increment
+          begin
+            rotator = KeyRotator::AttributeEncryption.new(user)
+            rotator.rotate
+            progress&.increment
+          rescue Encryption::EncryptionError => err # Don't use user.email in output...
```

Comment:
Why rescue from `Encryption::EncryptionError` and not all of `StandardError`?

Reply:
I originally had StandardError but switched it out at the last minute thinking we might have an even bigger problem and should probably halt if it's not an encryption error.  I also entertained putting in a task that simply does a dry run first.

Reply:
I wouldn't have a problem switching it back to StandardError.

Reply:
🤔, yeah I think that's what I'd do. We see plenty of errors related to encryption that are not strictly encryption errors.